File: 项目/OC项目/斗鱼(OC)/斗鱼(OC)/Classes/Category/NSDate+Extension.m
//
//  NSDate+Extension.m
//  斗鱼(OC)
//
//  Created by 金亮齐 on 2017/6/1.
//  Copyright © 2017年 醉看红尘这场梦. All rights reserved.
//


#import "NSDate+Extension.h"

@implementation NSDate (Extension)

+ (NSString *)nowDateString {
    NSDateFormatter *fmt = [[NSDateFormatter alloc] init];
    fmt.dateFormat = @"yyyy-MM-dd HH:mm";
    fmt.locale = [[NSLocale alloc] initWithLocaleIdentifier:@"zh"];
    return [fmt stringFromDate:[NSDate date]];
}

@end

File: 项目/Swift项目/微博/微博/Classes/Other/Tools/NSDate+Extension.swift
//
//  NSDate+Extension.swift
//  微博
//
//  Created by 金亮齐 on 2017/5/5.
//  Copyright © 2017年 醉看红尘这场梦. All rights reserved.
//

import Foundation

extension NSData {
    
    class func dateWithStr(time: String) ->NSDate {
        
        let formatter = NSDateFormatter()
        formatter.dateFormat = "EEE MMM d HH:mm:ss Z yyyy"
        formatter.locale = NSLocale(localeIdentifier: "en")
        let createdDate = formatter.dateFromString(time)!
        
        return createdDate
    }
    
    
}
